mreg_cli/util.py

```diff
@@ -411,7 +411,7 @@ def clean_hostname(name: typing.AnyStr) -> str:
     name = name.lower()
 
     # invalid characters?
-    if re.search("^([a-z0-9_][a-z0-9\-]*\.?)*$", name) is None:
+    if re.search("^(\*\.)?([a-z0-9_][a-z0-9\-]*\.?)*$", name) is None:
```

Comment:
Suggested change: `re.search("^(\*\.)?([a-z0-9_][a-z0-9\-]*\.?)+$", name)`
To force a non-empty match.

Reply:
Good point. Did it now in f104d7b